tracer/src/Datadog.Trace/Iast/Aspects/EntityFramework/EntityCommandAspect.cs

```diff
@@ -29,7 +30,9 @@ public static object ReviewSqlCommand(object command)
     {
         if (command is DbCommand dbCommand)
         {
-            IastModule.OnSqlQuery(dbCommand.CommandText, IntegrationId.SqlClient);
+            var commandText = dbCommand.CommandText;
```

Comment:
This could be null, is that being handled?

Reply:
It's handled. Still, reviewing it, it would be more efficient to check here. Thanks!